boards/x86/acrn/doc/index.rst

```diff
@@ -0,0 +1,123 @@
+.. _acrn:
+
+ACRN UOS (Guest Operating System)
+#################################
+
+Overview
+********
+
+This board configuration defines an ACRN User OS execution environment for
+running Zephyr RTOS applications.
+
+ACRN is a flexible, lightweight reference hypervisor, built with real-time
+and safety-criticality in mind, optimized to streamline embedded development
+through an open source platform. Check out the `Introduction to Project ACRN
+<https://projectacrn.github.io/latest/introduction/>`_ for more information.
+
+This baseline configuration can be used as a starting point for creating
+custom ACRN UOS configurations. It currently supports the following devices:
+
+* I/O APIC
+* local APIC timer
+* NS16550 UARTs
+
+Serial Ports
+------------
+
+The serial ports are assumed present at traditional ``COM1:`` and ``COM2:``
+I/O-space addresses (based at ``0x3f8`` and ``0x2f8``, respectively). Only
+polled operation is supported in this baseline configuration, as IRQ
+assignments under ACRN are configurable (and frequently non-standard).
+Interrupt-driven and MMIO operation are also possible.
+
+Building and Running
+********************
+
+#. The build process for ACRN targets is similar to other boards.  For example,
+you can build the :ref:`hello_world` sample for ACRN with:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: acrn
+   :goals: build
```

Comment:
Assuming we can also use `west` to build this, we should add this option `:tool: all`

Reply:
> Assuming we can also use `west` to build this, we should add this option `:tool: all`

I have just tested building the `hello_world` sample using `west` and it worked, let's add `:tool: all` to this section.

Reply:
done